torch/nn/functional.py

```diff
@@ -2815,6 +2815,26 @@ def multi_margin_loss(input, target, p=1, margin=1., weight=None, size_average=N
     torch.Size([1, 1, 12, 12])
 """)
 
+pixel_unshuffle = _add_docstr(torch.pixel_unshuffle, r"""
+pixel_unshuffle(input, downscale_factor) -> Tensor
+
+Rearranges elements in a tensor of shape :math:`(*, C, H \times r, W \times r)` to a
+tensor of shape :math:`(*, C \times r^2, H, W)`.
```

Comment:
Like the pixel shuffle documentation, this references `r` before it's defined. The input argument is "downscale_factor". Also, while this first statement is true, I think the highest order bit to communicate when describing this function is that it reverses the pixel shuffle operation, which can can be linked to in the first sentence. The identify can even be elaborated on in a mathematical line showing that unshuffle(shuffle(...)) is an identity. This first sentence is OK, and the reshaping performed is important, but it loses the context of what the reshape does, and there's no mention of how the values in the tensor are permuted. That is, an extremely cynical reading of this first sentence is simply that it randomly rearranges all the elements in the tensor and puts them into the new shape.